docs/environment.md

```diff
@@ -427,6 +431,33 @@ export GCM_MSAUTH_FLOW="devicecode"
 
 ---
 
+### GCM_MSAUTH_USEBROKER
+
+Use the operating system account manager where available.
+
+Defaults to the value `true`.
+
+Value|Description
+-|-
+`true` _(default)_|Use the operating system account manager as an authentication broker.
+`false`|Do not use the broker.
+
+##### Windows
+
+```batch
+SET GCM_MSAUTH_USEBROKER="false"
+```
+
+##### macOS/Linux
+
+```bash
+export GCM_MSAUTH_USEBROKER="false"
+```
+
+**Also see: [credential.msauthUseBroker](configuration.md#credentialmsauthusebroker)**
```

Comment:
Does the environment variable override the config setting, as is custom in Git?

Reply:
Yes. Take a look at aka.ms/gcmcore-env:

![image](https://user-images.githubusercontent.com/5658207/114196614-7173aa00-9949-11eb-958d-c2d6f5450c18.png)